tests/test_smoke.py

```diff
@@ -155,8 +155,9 @@ def test_region():
             f'sky launch -y -c {name} --region us-west-2 examples/minimal.yaml',
             f'sky exec {name} examples/minimal.yaml',
             f'sky logs {name} 1 --status',  # Ensure the job succeeded.
+            f'sky status --all | grep {name} | grep us-west-2',  # Ensure the region is correct.
         ],
-        f'sky down -y {name}',
+        f'sky down -y {name} {name}-2 {name}-3',
```

Comment:
Is this change required? Not sure how `{name}-2` and `{name}-3` clusters are being created. Same at L176